packages/foreman/nodejs-node-sass/nodejs-node-sass.spec

```diff
@@ -400,22 +399,25 @@ for tgz in %{sources}; do
 done
 %{?scl:end_of_scl}
 
-%setup -T -q -a 179 -D -n %{npm_cache_dir}
+%setup -T -q -a 177 -D -n %{npm_cache_dir}
 
 %build
 export SKIP_SASS_BINARY_DOWNLOAD_FOR_CI=true
 
 %{?scl:scl enable %{?scl_nodejs} - << \end_of_scl}
-npm install --ignore-scripts --cache-min Infinity --cache %{?scl:../}%{npm_cache_dir} --no-shrinkwrap --no-optional --global-style true %{npm_name}@%{version}
+npm install --ignore-scripts --cache-min Infinity --cache %{?scl:../}%{npm_cache_dir} --no-shrinkwrap --no-optional -
```

Comment:
Looks like the end of this accidentally got cut off.

Reply:
Since EL8 uses a newer NodeJS version, it needs to reference the npm cache the same way as SCL. I think at this point its safe to say we aren't packaging these for older NodeJS anymore. Therefore I would do the following:

Modify line 385 - 389 to be:

```
%define npm_cache_dir npm_cache
```

Then update this line to:

```
--cache ../%{npm_cache_dir}
```